### PR TITLE
fix(deploy-zip): update deploy-zip to include bonita-start/stop.sh

### PR DESCRIPTION
**Pull request by educhastenier**

### CODE REVIEW

File: deploy/distrib/src/main/resources/README.TXT [hunk 1]
```diff
@@ -56,12 +58,16 @@ Contents of this package:
...
 |-- license                                 			License files
 `-- README.txt                              			You are here!
 
 In order to install bonita you need to copy into the root folder of your application server:
 - the content of the directory matching it
-- for Jboss update the file bonita-platform-community.properties and bonita-tenant-community.properties to remove every occurrence of "java:comp/env/"
+- for Jboss update the files `platform-setup/platform_conf/initial/platform_engine/bonita-platform-community-custom.properties`
+  and `platform-setup/platform_conf/initial/tenant_template_engine/bonita-tenant-community-custom.properties`:
+      - uncomment lines containing "java:comp/env/"
+      - remove every occurrence of "java:comp/env/" (leave the rest of the value)
```
Comment (laurentleseigneur) [inline deploy/distrib/src/main/resources/README.TXT]: still quite unclear, but i finally understood what to do. 

thoses lines may be alreay present in  with specific comment like "# JBoss configuration: uncomment lines below to configure jndi"

due to build process, they should be commented in bonita-platform-community.properties / platform-setup/platform_conf/initial/tenant_template_engine/bonita-tenant-community.properties

File: deploy/distrib/src/main/resources/README.TXT [hunk 2]
```diff
@@ -56,12 +58,16 @@ Contents of this package:
 |   |   `-- lib
 |   |       `-- *.jar                    				Common libs that can be shared by several webapps (H2 listener)
 |   `-- welcome-content
-|       `-- favicon.ico
-|    
+|   |   `-- favicon.ico
+|   `-- bonita-start(.sh|.bat)                          Script used to start Bonita BPM Tomcat bundle
+|
 |-- platform-setup							            Platform creation/initialization files (Database scripts + configuration files)
```
Comment (laurentleseigneur) [inline deploy/distrib/src/main/resources/README.TXT]: my previous comment about naming this "setup" as in other distrib is still valid

File: deploy/distrib/src/main/resources/README.TXT (hunk 1 above)
Comment (educhastenier) [inline deploy/distrib/src/main/resources/README.TXT]: So I remove this paragraph, ok?

Comment (laurentleseigneur) [inline deploy/distrib/src/main/resources/README.TXT]: or refactor it and add jboss values commented in properties (engine PR)

Comment (educhastenier) [inline deploy/distrib/src/main/resources/README.TXT]: Ok, so let's do it in another PR

File: deploy/distrib/src/main/resources/README.TXT (hunk 2 above)
Comment (educhastenier) [inline deploy/distrib/src/main/resources/README.TXT]: Ok, maybe we'll do it, but in any case, it is out of the scope of this PR
